src/wallet/rpcwallet.cpp

```diff
@@ -4718,7 +4717,12 @@ UniValue walletcreatefundedpsbt(const JSONRPCRequest& request)
     CAmount fee;
     int change_position;
     CMutableTransaction rawTx = ConstructTransaction(request.params[0], request.params[1], request.params[2], request.params[3]);
-    FundTransaction(pwallet, rawTx, fee, change_position, request.params[4]);
+    // Make a copy and edit replaceability in-place
+    UniValue params_copy(request.params[4]);
+    if (!request.params[3].isNull()){
+        params_copy.pushKV("replaceable", request.params[3].get_bool());
```

Comment:
Suppose someone already has `replaceable` set in their options (because they copied it from `fundrawtransaction` or something). How does this interact with that?

Reply:
I'm not sure what's best in this case.

Reply:
Perhaps I can reverse my decision and only use the option's version.